packages/labs/context/src/lib/helper.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ContextRequestEvent} from './context-request-event.js';
+import {Context} from './create-context.js';
+
+/**
+ * A helper method for use in situations where a web component is isolated
+ * from the parent that provides the context.
+ *
+ * This helper provides static values for the specified contexts. It does
+ * not support updates via subscribe.
+ */
+export const provideHelper = (params: {[key: string]: unknown}) => {
```

Comment:
Here and elsewhere, the term "helper" is overly generic, like "util" or "meta." This does look like a helpful function, but most good functions are helpful, so naming it a helper doesn't aid the reader in understanding _how_ it's helpful, or in what situations.

Consider a name that communicates more. If I understand this right, I think `staticProvideHandler` is one candidate. `static` because it provides a constant value, and `handler` because it's used as an event handler.

Reply:
I would consider:
- `providerListener()` because they're event listeners, and this is a factory for making listeners that are providers.
- `contextProvider()` though it's a little generic compared to other ways of providing, it reads ok at the use site.
- `contextListener()` which might seem generic on the provide vs consume side, but only providers actually listen for events, while consumers fire them.

Reply:
Ah yeah, +1 to listener instead of handler.

Of those, `contextProvider` is my fave, followed by `providerListener`. I think `provide` is so key to what's happening that it needs to be in the name.